e2e/app/aou-elements/button.ts

```diff
@@ -27,12 +27,13 @@ export default class Button extends BaseElement {
   /**
    * Wait until button is clickable (enabled).
    */
-  async waitUntilEnabled(): Promise<JSHandle> {
-    const handle = this.element.asElement();
-    return await handle.evaluateHandle((e) => {
+  async waitUntilEnabled(): Promise<unknown> {
+    const result = await this.page.waitForFunction((e) => {
       const style = window.getComputedStyle(e);
       return style.getPropertyValue('cursor') === 'pointer';
-    }, this.element);
+    }, {}, this.element);
+    const jval = await result.jsonValue();
```

Comment:
Nit: I would leave out this variable, and just `return await result.jsonValue();`

If you do need the variable, I would use a more descriptive name than jval. Maybe `{rename of result}AsJson`?

Reply:
done